pkg/agent/openflow/pipeline.go

```diff
@@ -332,6 +332,7 @@ type client struct {
 	replayMutex   sync.RWMutex
 	nodeConfig    *config.NodeConfig
 	networkConfig *config.NetworkConfig
+	egressConfig  *config.EgressConfig
```

Comment:
Do we still need this?

Reply:
Thanks for your mention, I will remove this parameter.